weather_mv/loader_pipeline/ee.py

```diff
@@ -756,18 +757,40 @@ def start_ingestion(self, asset_data: AssetData) -> t.Optional[str]:
                         raise ee.EEException(response.text)
 
                     return asset_name
+                
                 else:  # as a COG based image.
-                    ee.data.createAsset({
-                        'name': asset_name,
-                        'type': self.ee_asset_type,
-                        'gcs_location': {
-                            'uris': [asset_data.target_path]
-                        },
-                        'startTime': asset_data.start_time,
-                        'endTime': asset_data.end_time,
-                        'properties': asset_data.properties,
-                    })
+                    response = session.post(
+                        url=(
+                            f'https://earthengine.googleapis.com/v1alpha/projects/{self.get_project_id()}/'
```

Comment:
If we have importExternal endpoint for `earthengine-highvolume.googleapis.com` api then we should use that.
```suggestion
                            f'https://earthengine-highvolume.googleapis.com/v1/projects/{self.get_project_id()}/'
```

Reply:
- we have endpoint for earthengine-highvolume.googleapis.com API, but cannot find v1 for that, so keeping v1alpha for now.